test/python/visualization/test_circuit_matplotlib_drawer.py

```diff
@@ -65,6 +65,7 @@ def test_empty_circuit(self):
 
     @unittest.skipIf(not visualization.HAS_MATPLOTLIB,
                      'matplotlib not available.')
+    @unittest.skip('This does not work on OSX')
```

Comment:
While I agree these tests are just a pain to keep working instead of unconditionally skipping them here maybe we should make this conditional on sys.platform == 'darwin'. Or at least change the skip message to just say it's unreliable due to system differences or something. It'll just be weird to see a skip message that it doesn't work on osx when running on linux.

Moving forward I'd really like to get rid of all these image comparison tests. They're more trouble than they're worth. We're more testing our ability toreliably configure matplotlib (and latex for those drawer tests) and less actually verifying functionality of the drawers.